realm/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -35,28 +35,27 @@
 
 /**
  * A RealmConfiguration is used to setup a specific Realm instance.
- *
- * Instances of a RealmConfiguration can only created by using the {@link io.realm.RealmConfiguration.Builder} and calling
- * its {@link io.realm.RealmConfiguration.Builder#build()} method.
- *
- * A commonly used RealmConfiguration can easily be accessed by first saving it as
- * {@link Realm#setDefaultConfiguration(RealmConfiguration)} and then using {@link io.realm.Realm#getDefaultInstance()}.
- *
+ * <p/>
+ * Instances of a RealmConfiguration can only created by using the {@link io.realm.RealmConfiguration.Builder} and
+ * calling its {@link io.realm.RealmConfiguration.Builder#build()} method.
+ * <p/>
+ * A commonly used RealmConfiguration can easily be accessed by first saving it as {@link
+ * Realm#setDefaultConfiguration(RealmConfiguration)} and then using {@link io.realm.Realm#getDefaultInstance()}.
+ * <p/>
  * A minimal configuration can be created using:
- *
+ * <p/>
  * {@code RealmConfiguration config = new RealmConfiguration.Builder(getContext()).build())}
- *
- * This will create a RealmConfiguration with the following properties
- * - Realm file is called "default.realm"
- * - It is saved in Context.getFilesDir()
- * - It has its schema version set to 0.
+ * <p/>
+ * This will create a RealmConfiguration with the following properties - Realm file is called "default.realm" - It is
+ * saved in Context.getFilesDir() - It has its schema version set to 0.
  */
-public class RealmConfiguration {
+public final class RealmConfiguration {
```

Comment:
Yes... some nice rules about utility classes. They should not be instantiable and they should be final.